### DISCUSSION TITLE
Just wanne say - Yeeeah, thank you

### DISCUSSION THREAD

**Discussion by habnefrage** [rdmtc/RedMatic#465] [General]
The solution is awsome and works out of the box.
Very good idea, excelent implementation, THANK YOU!

**Comment by hobbyquaker** [comment]
Danke! RedMatic lebt weiter - RedMatic 9 (Node.js 24, Node-RED 5) ist als Vorabversion verfügbar: https://github.com/rdmtc/RedMatic/releases

---
*Written by Claude Fable on behalf of hobbyquaker.*
